contents/docs/feature-flags/snippets/faq-false-or-none-events.mdx

```diff
@@ -1,5 +1,7 @@
-## My `Feature Flag Called` events show `None`, `(empty string)`, or `false` instead of my variant names
+## My feature flag called events don't show my variant names
```

Comment:
```suggestion
## My feature flag-called events don't show my variant names
```

Reply:
"feature flag called" is a type of event, hence the name.

Reply:
Ah, in that case, `feature flag called` might be worth sticking with!

Reply:
Keeping the backticks in the title looks really bad which is why I made this change.